src/components/animals/ProfileDropzone.js

```diff
@@ -1,32 +1,61 @@
-import React, { PropTypes } from 'react';
+import React, { PropTypes, Component } from 'react';
 import Dropzone from 'react-dropzone';
 import '../../styles/images-dropzone.scss';
 
-const ProfileDropzone = ({ profilePic, onDrop, error }) => {
+class ProfileDropzone extends Component {
+  constructor(props, context) {
+    super(props, context);
+    this.state = {
+      click: false
+    };
+    this.onOpenClick = this.onOpenClick.bind(this);
+  }
+
+  onOpenClick() {
+    const { click } = this.state;
+    if (!click) {
+      this.dropzone.open();
+      this.setState({ click: true });
+      let that = this;
```

Comment:
esto se puede hacer un poco mas prolijo y eficiente(evitas pasarle todo el `this` a la funcion) : 
```
const setClick = () => {
  this.setState({ click: false });
};
setTimeout(setClick, 1000);
```